tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """
+
+.. include:: ../../doc/links.inc
```

Comment:
just `../../links.inc` should be correct, look at `git grep "links\.inc"` for other tutorials